dashboard/app/helpers/ai_system_prompts/evaluate_system_prompt_helper.rb

```diff
@@ -22,7 +23,20 @@ def self.get_system_prompt(level, unit, evaluation_type)
   # we should update this method. Right now we're just defaulting to assessing against the level's instructions
   # or very generic criteria for programming levels.
   def self.get_evaluation_criteria(level)
-    if level.upper_grades_programming_level?
+    skills_based_evaluation_criteria = {
+      22595 => [
+        "Are all of the variables in the student's code named conventionally?",
+        "Is every variable declared before it is used?",
+      ],
```

Comment:
I'm kind of wondering how different the skill description and evaluation criteria are... Like if the skill is "Names variables using appropriate conventions". Would the evaluation criteria just be: 
`Is there evidence in the code sample that the student ${skill.description}`

If we can standardize on verbiage, it might be easier.